manifests/globals.pp

```diff
@@ -122,13 +122,13 @@
 
   Optional[String[1]] $validcon_script_path        = undef,
 
-  Optional[Variant[String[1], Stdlib::Absolutepath]] $initdb_path                 = undef,
-  Optional[Variant[String[1], Stdlib::Absolutepath]] $psql_path                   = undef,
-  Optional[Variant[String[1], Stdlib::Absolutepath]] $pg_hba_conf_path            = undef,
-  Optional[Variant[String[1], Stdlib::Absolutepath]] $pg_ident_conf_path          = undef,
-  Optional[Variant[String[1], Stdlib::Absolutepath]] $postgresql_conf_path        = undef,
+  Optional[Stdlib::Absolutepath]                     $initdb_path                 = undef,
```

Comment:
I'd not apply any alignment between data types and variables, but that's just me.